tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -416,29 +416,28 @@ func (i InstantiateCCUIObject) approveCCusingCLI(instantiateObject InstantiateCC
 
 //commitCCusingCLI -- committing cc using CLI
 func (i InstantiateCCUIObject) commitCCusingCLI(instantiateObject InstantiateCCUIObject) error {
-
-	orgName := instantiateObject.ChannelOpt.OrgName[0]
-	var connProfilePath string
-	if strings.Contains(instantiateObject.ConnProfilePath, ".yaml") || strings.Contains(instantiateObject.ConnProfilePath, ".json") {
-		connProfilePath = instantiateObject.ConnProfilePath
-	} else {
-		connProfilePath = fmt.Sprintf("%s/connection_profile_%s.yaml", instantiateObject.ConnProfilePath, orgName)
+	orgNameForOrderer := instantiateObject.ChannelOpt.OrgName[0]
```

Comment:
I like that you are more specific about which orgName, but In the spirit of shorter names, how about ordererOrgName?
Same for each of the below variables...

Reply:
Sure, I can change that